public/app/features/plugins/admin/components/Badges/PluginUpdateAvailableBadge.tsx

```diff
@@ -12,13 +12,7 @@ type Props = {
 
 export function PluginUpdateAvailableBadge({ plugin }: Props): React.ReactElement | null {
   const styles = useStyles2(getStyles);
-
-  // Currently renderer plugins are not supported by the catalog due to complications related to installation / update / uninstall.
-  if (plugin.hasUpdate && !plugin.isCore && plugin.type !== PluginType.renderer) {
-    return <p className={styles.hasUpdate}>Update available!</p>;
-  }
-
-  return null;
+  return <p className={styles.hasUpdate}>Update available!</p>;
 }
```

Comment:
I really liked that you moved this from this component to the call site for better overview!